### DISCUSSION TITLE
리트코드 스터디 정기 모임 11주차 공지

### DISCUSSION THREAD

**Discussion by SamTheKorean** [DaleStudy/leetcode-study#544] [공지사항]
스터디 2기 참여자 여러분, 안녕하세요. Sam입니다👋

열 한번째 모임도 전 모임과 같이 지원자를 받고 각각 문제에 대해서 설명 및 논의하는 시간을 가지겠습니다!

문제당 한 분씩 풀이를 할 예정이고, 신청은 선착순으로 각 주차에 있는 아직 풀이 희망자가 없는 문제 중 원하시는 문제와 디스코드 명을 one-bit-integer / Sam 과 같은 형식으로 댓글로 남겨주시면 됩니다!ㅎㅎ

추가적으로 금주에 받은 좋은 pr 리뷰에 대해서 이야기해보는 시간을 가질 예정입니다ㅎㅎ 다른 스터디원분께 받은 도움이 됐던 PR이 있다면 댓글로 링크 공유 부탁드리겠습니다!

다음 모임도 이전과 마찬가지로 아래 링크로 RSVP 부탁드리겠습니다! (https://discord.gg/WeaZuFYv?event=1297415907429060608)

**Comment by DaleSeo** [comment]
@SamTheKorean RSVP 링크가 깨진 것 같아요 😱

**Reply by SamTheKorean** [reply]
업데이트 했습니다! 피드백 감사합니다! @DaleSeo

**Comment by TonyKim9401** [comment]
Reorder List / Tony

**Reply by TonyKim9401** [reply]
발표자료 링크 입니다.
https://docs.google.com/presentation/d/1Ake8V_r2-NNzuqupc55YGZMTnShr17vDA7uxzlwgtqY/edit#slide=id.g30ed68c4347_0_56

**Comment by DaleSeo** [comment]
@HC-kang Graph Valid Tree https://github.com/DaleStudy/leetcode-study/pull/546#discussion_r1817851454
